cppsrc/U8x8lib.h

```diff
@@ -128,6 +128,12 @@
 #endif
 #endif
 
+/*ISSUE #991 */
+#ifdef BOARD_NR_SPI
```

Comment:
Why does the ESP32 Arduino port use BOARD_NR_SPI? It should better use SPI_INTERFACES_COUNT instead.
From my perspective this should be changed in the ESP32 software.

Reply:
I know, the lib uses this and not SPI_INTERFACES_COUNT. Is the only way i can figure out the problem.